### DISCUSSION TITLE
When one of my submitters of Code has died and I want to remove him from my team!

### DISCUSSION THREAD

**Discussion by ITI-IoT** [community/community#36987] [Other Feature Feedback, Questions, & Ideas]
When a person has died it will be natural to remove him from my team. Will this have any influence on his submitted stuff?

**Comment by FatChicken277** [comment]
No, just imagine what would happen if you want to fire an employee from your company, his job remains intact, but obviously he will lose access to the project, so he may be left with a local copy that cannot be updated. [check this](https://docs.github.com/en/github-ae@latest/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization)
